Lib/_pyrepl/readline.py

```diff
@@ -161,6 +170,11 @@ def get_completions(self, stem: str) -> list[str]:
             result.sort()
         return result
 
+    def get_module_completions(self) -> list[str]:
+        completer = ModuleCompleter(namespace={'__package__': '_pyrepl'})  # TODO: namespace?
```

Comment:
One thing I wasn't sure about is how to handle relative imports in regards to `__package__`. In PyREPL, it is set to `_pyrepl` so I replicate that here but maybe we want it to be configurable?